Functions.js

```diff
@@ -31,3 +31,12 @@ app.delete('/delete/user/:id', (req, res) => {
     res.send(user)
 	//bug fixed
  })
+
+app.post('/add/project', (req, res) => {
+    const newId = projects.map(u => u.id).reduce(function(a, b) { return Math.max(a, b) }) + 1;
+    let newProject = req.body
+    newProject.id = newId
+    projects.push(newProject)
+    console.log(JSON.stringify(req.body))
```

Comment:
this line should not be into the release version

Reply:
all done